client/src/components/EditProfile/ProfilePhoto/useStyles.ts

```diff
@@ -3,19 +3,20 @@ import { createStyles, makeStyles, Theme } from '@material-ui/core';
 const useStyles = makeStyles((theme: Theme) =>
   createStyles({
     root: {
-      '& > *': {
-        margin: theme.spacing(1),
-      },
       display: 'flex',
+      flexDirection: 'column',
       justifyContent: 'center',
       alignItems: 'center',
+      '& .MuiButton-label': {
+        textTransform: 'none',
+      },
```

Comment:
Nice, this is a perfectly valid way to revert the uppercase text transformation! 

However, apart from the auth buttons in the mocks, none of the buttons have uppercase text transformation. Thus if we were able to change the default behaviour of Buttons throughout the whole app, it would save us from applying this style over and over again. This is exactly why having a theme is so useful material ui! 

I recommend giving this another try but try to [customize the theme](https://v4.mui.com/customization/theming/#theming) this time. Remember, you can check out the links I mentioned [here](https://github.com/hatchways/team-reindeer/pull/61#pullrequestreview-833670563) for help. I'll still leave this a bonus exercise though so you can leave it for another time 👍🏼

Reply:
@sholabioku As Ugur mentioned try adding the relevant override to MUI's theme :)